include/openssl/evp.h

```diff
@@ -109,6 +109,9 @@ int (*EVP_MD_meth_get_ctrl(const EVP_MD *md))(EVP_MD_CTX *ctx, int cmd,
 /* digest can only handle a single block */
 #  define EVP_MD_FLAG_ONESHOT     0x0001
 
+/* digest is extensible-output finction, XOF */
```

Comment:
Missed 1cm to the left. Thanks! Fixed. Since it's pure commentary update I consider that approval still holds...